modules/launcher/src/main/resources/COPYRIGHT.txt

```diff
@@ -0,0 +1,4 @@
+WSO2 Message Broker
+Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shouldn't it be 2018?

Reply:
No Since we started development in 2017. :smile: